retrospective-update/functions.py

```
@@ -0,0 +1,351 @@
+
+    Raises:
+        Exception: If the sync command fails.
+    """
+    # sync the zarrs. We can use sync because 0.* files are not is not on local side
+    for zarr, s3_zarr in zip([local_hourly_zarr, local_daily_zarr], [s3_hourly_zarr, s3_daily_zarr]):
+        CL.ping('RUNNING', f'syncing-{zarr}-to-S3')
+        result = subprocess.run(
+            f"s5cmd --credentials-file {credentials} sync {zarr}/ {s3_zarr}/",
+            shell=True, capture_output=True, text=True,
+        )
+        if not result.returncode == 0:
```

Ruff (0.12.2)

Use `result.returncode != 0` instead of `not result.returncode == 0`

Replace with `!=` operator

(SIM201)